app/plugin/FeatureInfoWindow.js

```diff
@@ -0,0 +1,136 @@
+Ext.define('CpsiMapview.plugin.FeatureInfoWindow', {
+    extend: 'Ext.plugin.Abstract',
+    alias: 'plugin.cmv_feature_info_window',
+    pluginId: 'cmv_feature_info_window',
+
+    requires: [
+        'Ext.Component',
+        'CpsiMapview.view.window.MinimizableWindow',
+        'BasiGX.view.grid.FeaturePropertyGrid',
+        'BasiGX.util.Map'
+    ],
+
+    init: function () {
+        var me = this;
+
+        var mapComp = BasiGX.util.Map.getMapComponent();
+        var map = mapComp.getMap();
+
+        map.on('singleclick', function (evt) {
+            me.requestFeatureInfos(mapComp, evt);
+        });
+    },
+
+    requestFeatureInfos: function (mapComp, evt) {
+        var me = this;
+
+        var map = mapComp.getMap();
+
+        var layers = [];
+        var resolution = map.getView().getResolution();
+        var projection = map.getView().getProjection();
+
+        var format = new ol.format.GeoJSON();
+
+        map.forEachLayerAtPixel(evt.pixel, function (layer) {
+            layers.push(layer);
+        }, undefined, function (layer) {
+            return layer.get('featureInfoWindow');
+        });
+
+        var window = me.openFeatureInfoWindow(map, evt);
```

Comment:
Maybe use win rather than window (the global variable) to avoid confusion when looking at the code in the future?

Reply:
Agreed